alphagov/paas-team-manual
# Make style improvements

What
----

Have made some style improvements for clarity and better accessibility. 

How to review
-------------

1. Preview locally and make sure it renders correctly.
2. Check that the content is still accurate and up-to-date.
3. Check that the content accurately represents the information given in the diagram. If anything is missing or incomplete, please suggest an improvement. 

Who can review
--------------

Not @mcnalc
